src/scaffoldmaker/meshtypes/meshtype_3d_stellate1.py

```diff
@@ -105,45 +166,43 @@ def generateBaseMesh(cls, region, options):
         :param options: Dict containing options. See getDefaultOptions().
         :return: None
         """
-        isDefault = 'Default' in options['Base parameter set']
-        isMouse = 'Mouse' in options['Base parameter set']
-
-        armCount = 3
-        elementLengthCentral = options['Element width central']
-        elementLengths = [options['Element length along arm'],
-                 options['Element width across arm'],
-                 options['Element thickness']]
-        elementsCount1 = options['Numbers of elements along arms']
-        elementsCount2 = 2
-        elementsCount3 = 1
-        useCrossDerivatives = False
+        parameterSetName = options['Base parameter set']
+        isDefault = 'Default' in parameterSetName
+        isMouse = 'Mouse' in parameterSetName and 'Mean' not in parameterSetName
+        isMeanMouse = 'Mean mouse' in parameterSetName
 
         fm = region.getFieldmodule()
         nodes = fm.findNodesetByFieldDomainType(Field.DOMAIN_TYPE_NODES)
         coordinates = findOrCreateFieldCoordinates(fm)
-
+        mesh = fm.findMeshByDimension(3)
+        cache = fm.createFieldcache()
         nodetemplate = nodes.createNodetemplate()
         nodetemplate.defineField(coordinates)
         nodetemplate.setValueNumberOfVersions(coordinates, -1, Node.VALUE_LABEL_VALUE, 1)
         nodetemplate.setValueNumberOfVersions(coordinates, -1, Node.VALUE_LABEL_D_DS1, 1)
         nodetemplate.setValueNumberOfVersions(coordinates, -1, Node.VALUE_LABEL_D_DS2, 1)
-        if useCrossDerivatives:
-            nodetemplate.setValueNumberOfVersions(coordinates, -1, Node.VALUE_LABEL_D2_DS1DS2, 1)
-        mesh = fm.findMeshByDimension(3)
-        cache = fm.createFieldcache()
 
-        markerGroup = findOrCreateFieldGroup(fm, "marker")
-        markerName = findOrCreateFieldStoredString(fm, name="marker_name")
-        markerLocation = findOrCreateFieldStoredMeshLocation(fm, mesh, name="marker_location")
-
-        markerPoints = findOrCreateFieldNodeGroup(markerGroup, nodes).getNodesetGroup()
-        markerTemplateInternal = nodes.createNodetemplate()
-        markerTemplateInternal.defineField(markerName)
-        markerTemplateInternal.defineField(markerLocation)
+        armCount = 3
+        elementLengthCentral = options['Element width central']
+        elementLengths = [options['Element length along arm'],
+                          options['Element width across arm'],
+                          options['Element thickness']]
+        elementsCount1 = options['Numbers of elements along arms']
```

Comment:
This should be called elementsCountsAlongArms; elementsCount1 sounds like a single number.